e2e-tests/tests/e2e_arknote.rs

```diff
@@ -0,0 +1,67 @@
+#![allow(clippy::unwrap_used)]
+
+use ark_core::ArkNote;
+use bitcoin::key::Secp256k1;
+use bitcoin::Amount;
+use common::init_tracing;
+use common::set_up_client;
+use common::Nigiri;
+use std::sync::Arc;
+
+mod common;
+
+#[tokio::test]
+#[ignore]
+pub async fn e2e_arknote_redemption() {
+    init_tracing();
+    let nigiri = Arc::new(Nigiri::new());
+
+    let secp = Secp256k1::new();
+
+    // Create a test wallet
+    let (alice, _) = set_up_client("alice".to_string(), nigiri.clone(), secp.clone()).await;
+    let alice_offchain_address = alice.get_offchain_address().unwrap().0;
+
+    tracing::info!(
+        ?alice_offchain_address,
+        "Created Alice's wallet for ArkNote redemption test"
+    );
+
+    // Create a test ArkNote
+    let fund_amount = Amount::from_sat(1000);
+    let arknote = alice.make_arknote(fund_amount).await.unwrap();
```

Comment:
Ah okay, you are creating a test note. I mean this is useful for tests, but can't we get an legitimate Ark note from the server instead? This would save having to add the admin client just for testing.